### PR TITLE
jq => 1.7

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=jq17 crew update
```

### CODE REVIEW

File: packages/jq.rb [hunk 1]
```diff
@@ -1,45 +1,50 @@
...
-    system "CC=clang LD=ld.lld  CFLAGS='-flto -pipe -O3 -fuse-ld=lld -static' \
-      CXXFLAGS='-flto -pipe -O3 -static' \
-      LDFLAGS='-flto -static' \
-      ./configure \
-      --prefix=#{CREW_PREFIX} \
-      --libdir=#{CREW_LIB_PREFIX} \
-      --mandir=#{CREW_MAN_PREFIX} \
-      --enable-all-static \
+  depends_on 'glibc' unless ARCH == 'i686'
+  depends_on 'llvm17_dev' => :build if ARCH == 'i686' # has a libm issue building againsg glibc.
```
Comment (uberhacker) [inline packages/jq.rb:27]: Combine into one line?
```
depends_on (ARCH == 'i686') ? 'llvm17_dev' : 'glibc';
```

Comment (satmandu) [reply packages/jq.rb:27]: The llvm17_dev is only a build dep though...

**Review by uberhacker — APPROVED** [review]
Definitely cleaner.

File: packages/jq.rb [hunk 2]
```diff
@@ -1,45 +1,30 @@
...
-      --disable-maintainer-mode \
-      --disable-docs " # there's no support for manpages
-    system 'make'
-  end
+  depends_on 'glibc'
 
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-  end
+  configure_options '--disable-maintainer-mode \
```
Comment (Zopolis4) [inline packages/jq.rb:28]: `jq` has maintainer mode [disabled by default](https://github.com/jqlang/jq/blob/master/configure.ac#L12).

Additionally, I believe it does generate manpages as part of its documentation. https://github.com/jqlang/jq/tree/master/docs#documentation

**Pull request merged**